lib/views/pages/home/top_bar/home_top_bar.dart

```diff
@@ -0,0 +1,37 @@
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+
+/// This widget is the top bar of the home page
+/// It contains the timeline filters and the user profile picture
+class HomeTopBar extends HookConsumerWidget {
+  static const homeTopBarKey = Key("homeTopBar");
+  static const profilePictureKey = Key("profilePicture");
+
+  static const titleText = "Proxima";
+
+  const HomeTopBar({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final title = Text(
+      titleText,
+      style: Theme.of(context).textTheme.headlineMedium,
+    );
+
+    const userAvatar = CircleAvatar(
+      key: profilePictureKey,
+      child: Text("PR"),
```

Comment:
This needs to be personalized (not a hardcode like here)

Reply:
This is a static overview UI, this should be implemented along with the view model.

Reply:
This is currently a static overview UI. I am letting flexibility to the person who will implement the view model.